.github/workflows/deploy_ios_release_firebase.yml

```
@@ -8,7 +8,7 @@ name: Deploy iOS Release Build To Firebase
 
 on:
   push:
-    branches: [ develop, main ]
+    branches: [ develop, main, bug/cicd-* ]
```

You can remove this test now
```suggestion
    branches: [ develop, main ]
```